projects/storefrontlib/cms-components/misc/message/message.component.ts

```diff
@@ -67,4 +80,15 @@ export class MessageComponent {
         return ICON_TYPE.SUCCESS;
     }
   }
+
+  ngAfterViewInit(): void {
+    // TODO: (CXSPA-7725) - Remove feature flag next major release.
```

Comment:
We no longer need to make these TODO comments ;)

Reply:
Comments removed